docs/public/static/schemas/unversioned/eas-json-build-common-schema.js

```diff
@@ -66,7 +66,7 @@ export default [
     enum: ['default', 'medium', 'large'],
     description: [
       'The resource class that will be used to run this build.',
-      'To see mapping for each platform, see [Android-specific resource class field](eas-json/#resourceclass-1) and [iOS-specific resource class field](eas-json/#resourceclass-2) documentation.',
+      'To see mapping for each platform, see [Android-specific resource class field](json/#resourceclass-1) and [iOS-specific resource class field](json/#resourceclass-2) documentation.',
```

Comment:
```suggestion
      'To see mapping for each platform, see [Android-specific resource class field](#resourceclass-1) and [iOS-specific resource class field](#resourceclass-2) documentation.',
```

Just a small nit since there is no need to use `json/` in the link's path since the both resourceClass definitions exist on the same page.